xmtp-inbox-ios/Views/ConversationCellView.swift

```diff
@@ -45,4 +54,16 @@ struct ConversationCellView: View {
 			}
 		}
 	}
+
+	var messagePreview: String {
+		do {
+			guard let message = mostRecentMessage else {
```

Comment:
This could be:

```suggestion
			guard let mostRecentMessage else {
```

instead of assigning a new variable.